### PR TITLE
MEN-2335 Update vendored dependency net/http2 to latest version

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: LIC_FILES_CHKSUM.sha256 [hunk 1]
```diff
@@ -28,3 +28,6 @@ ffa15bdce332058a03a1d923910864fb6e58bf6df66a0e3914284725b327183e  vendor/github.
 #
 # OpenLDAP Public License
 310fe25c858a9515fc8c8d7d1f24a67c9496f84a91e0a0e41ea9975b1371e569  vendor/github.com/bmatsuo/lmdb-go/LICENSE.mdb.md
+#
+# Unknown license
+2d36597f7117c38b006835ae7f537487207d8ec407aa9d9980794b2030cbc067  vendor/golang.org/x/text/LICENSE
```
Comment (lluiscampos) [inline LIC_FILES_CHKSUM.sha256]: I couldn't figure out the actual license. Any suggestion?

Comment (kacf) [reply LIC_FILES_CHKSUM.sha256]: This is BSD-3-Clause.

File: Makefile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 GO ?= go
 GOFMT ?= gofmt
 V ?=
-PKGS = $(shell go list ./... | grep -v vendor)
+PKGS = $(shell go list ./... | grep -v vendor | grep -v test)
```
Comment (kacf) [inline Makefile]: This has been [fixed and just merged](https://github.com/mendersoftware/mender/commit/51aea56a59c9698eb0639dec3cbacf454a4faa15) a couple of minutes ago, so you can drop this patch.

Comment (lluiscampos) [reply Makefile]: :+1:

**Pull request merged**